data/lib/core/container.lua

```diff
@@ -1,3 +1,44 @@
 function Container.isContainer(self)
 	return true
 end
+
+function Container.createLootItem(self, item)
```

Comment:
You need to check in here first if container is full, else it'll throw the warning message that it cannot add the item to the container.
```lua
if self:getEmptySlots() == 0 then
        return true
end
```

Reply:
thanks @EvilHero90